lib/whitelistReader.js

```diff
@@ -0,0 +1,16 @@
+const fs = require(`fs`);
+const Pnr = require(`./pnr`);
+
+class WhitelistReader {
+  static fetch(rootPath) {
+    const whiteListPath = `${rootPath}/.pnr-whitelist`;
```

Comment:
Should probably build path with `path.join` instead of hard coding folder separator. I.e. https://nodejs.org/api/path.html#path_path_join_paths

Reply:
Fixed!